salt/example.top

```diff
@@ -34,7 +35,7 @@ base:
         - elife.hub
         #- elife.init # this is 'elife' at the very top
         # See https://github.com/elifesciences/jira-import/issues/4360
```

Comment:
that link is outdated (redirects though)

Reply:
fixed